src/main/java/emissary/output/roller/ICoalescer.java

```diff
@@ -0,0 +1,28 @@
+package emissary.output.roller;
+
+import emissary.roll.Rollable;
```

Comment:
Remove unused import

Reply:
done

Reply:
FYI, my [impsort-maven-plugin](https://github.com/revelc/impsort-maven-plugin) will remove these automatically in the build.